palettes/InkStitch Isacord Polyester.gpl

```diff
@@ -2,394 +2,402 @@ GIMP Palette
+ 12 148 135	       	        Deep Sea Blue	  4442
+ 30 194 166	       	           Truly Teal	  4452
+ 51 102 102	       	               Spruce	  4515
+ 51 153 153	       	            Caribbean	  4531
+ 47 185 165	       	            Deep Aqua	  4610
+ 51 204 180	       	                 Jade	  4620
+    0 89 81	       	             Seagreen	  4625
+ 75 120 125	       	               Amazon	  4643
+102 130 115	       	              Mallard	  4644
+136 213 198	       	           Aquamarine	  4740
+133 171 167	       	         Vintage Blue	  4752
+147 206 200	       	          Mystic Blue	  4952
+    0 96 86	       	          Rain Forest	  5005
+  0 153 135	       	         Scotty Green	  5010
+220 255 230	       	               Luster	  5050
+  0 165 120	       	                Green	  5100
+ 51 175 153	       	            Dark Jade	  5101
+ 51 204 165	       	       Baccarat Green	  5115
+ 51 204 153	       	        Trellis Green	  5210
+160 204 180	       	          Silver Sage	  5220
+153 240 204	       	         Bottle Green	  5230
+  38 102 89	       	          Field Green	  5233
+    2 73 48	       	         Bright Green	  5324
+   0 115 80	       	            Evergreen	  5326
+    0 73 63	       	                Swamp	  5335
+    0 60 20	       	         Forest Green	  5374
+   0 135 81	       	             Shamrock	  5411
+   0 122 61	       	          Irish Green	  5415
+ 60 175 120	       	            Swiss Ivy	  5422
+129 223 149	       	                 Mint	  5440
+166 216 177	       	        Basic Seafoam	  5450
+  28 206 40	       	             Limedrop	  5500
+  51 158 53	       	              Emerald	  5510
+   0 158 15	       	                 Ming	  5513
+   0 164 90	       	                Kelly	  5515
+153 204 102	       	                 Pear	  5531
+100 130 112	       	          Garden Moss	  5542
+158 170 153	       	            Palm Leaf	  5552
+   35 79 51	       	           Deep Green	  5555
+   24 56 35	       	    Enchanting Forest	  5565
+180 240 130	       	          Bright Mint	  5610
+   0 183 96	       	          Light Kelly	  5613
+ 102 153 51	       	                 Lime	  5633
+   33 91 51	       	           Green Dust	  5643
+225 255 204	       	         Spring Frost	  5650
+119 145 130	       	               Willow	  5664
+  82 108 81	       	            Asparagus	  5743
+175 204 130	       	                 Kiwi	  5822
+220 255 185	       	           Chartreuse	  5830
+ 156 189 84	       	               Celery	  5832
+ 153 190 51	       	             Limabean	  5833
+    90 60 0	       	           Herb Green	  5866
+  158 196 0	       	           Erin Green	  5912
+ 140 160 90	       	          Grasshopper	  5933
+ 153 153 51	       	           Moss Green	  5934
+ 217 255 66	       	      Sour Apple neon	  5940
+  80 110 51	       	       Backyard Green	  5944
+ 255 255 63	       	    Mountain Dew neon	  6010
+190 231 110	       	             Tamarack	  6011
+ 218 224 86		                  Limelight	  6031
```

Comment:
wrong identation

Reply:
Yeah it turns out the indentation doesn't actually matter too much, it should still work.

We did our best to line them up, but there were some edge cases where it didn't align.